src/tests/stdlib/memmove_test.c

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file memmove_test.c
+ *
+ * @date June 29, 2020
+ * @author: KOUAM Ledoux
+ */
+
+#include <string.h>
+#include <embox/test.h>
+
+EMBOX_TEST_SUITE("memmove test suit");
+
+TEST_CASE("move full data")
```

Comment:
please use code-style brackets 
Must be 
```
TEST_CASE("move full data") {
```

Reply:
Okay , ...